### ISSUE TITLE
Power button switch to art mode does not work

### ISSUE THREAD

**Issue by ehn** [ollo69/ha-samsungtv-smart#295]
**Describe the bug**
Setting the option _power button switch to art mode_ does not work. The TV just turns off.

However, calling the `samsungtv_smart.set_art_mode` service does correctly put the TV in art mode.

**Expected behavior**
When using the power-off button in the UI, the TV should switch to art mode.

**Environment details:**
 - Environment (HASSIO, Raspbian, etc): Home Assistant Core running as a Docker image
 - Home Assistant version installed: 2023.12.3
 - Component version installed: v0.13.1
 - Last know working version:
 - TV model: QE65LS03AAUXXC (The Frame from 2021)

**Output of HA logs**
There is nothing related to the integration in the logs.

**Comment by johaywood** [comment]
Same issue here -- running the same versions of HA and component in Docker. TV model is `QN32LS03TBFXZA`. Nothing in the logs, but the TV just powers off when using the power button when config is set to switch to art mode. `samsungtv_smart.set_art_mode` works fine.

**Comment by ehn** [comment]
The issue has not been resolved as far as I can tell.

**Comment by deadrubberboy** [comment]
Same issue here. I've never had success getting this or the standard Samsung integrations to work with Art Mode. I refuse to use SmartThings and have my TV blocked at my router. That said, it IS plugged in RJ45 and I can turn it on/off etc. Lastly, I have a HomeBridge plug in (for HomeKit) that toggles Art Mode perfectly. It's the last thing still in HomeBridge, since I went all in on Home Assistant. If I can get the Frame Art Mode working I ditch Home Bridge finally. 

My desired state (works in HK w/ the HB plug in) is an automation that goes "When Apple TV Turns Off (this powers off the Frame) turn Art Mode ON).". Basically when I stop watching TV - Put in Art Mode.

**Comment by ehn** [comment]
The issue has not been resolved as far as I can tell.

**Comment by ehn** [comment]
The issue has not been resolved as far as I can tell.

**Comment by ehn** [comment]
The issue has not been resolved as far as I can tell.

**Comment by deadrubberboy** [comment]
Agreed.  I ended up reinstalling  Homebridge while we wait.  The Tizen integration there allows an automation that works “trigger -tv off/action -art mode on” via HomeKit.  Thought I was 100% Home Assistant, this is the last piece.

**Comment by ehn** [comment]
The issue has not been resolved as far as I can tell, and shouldn't be considered stale.

**Comment by deadrubberboy** [comment]
Agreed

**Comment by hcurtiss** [comment]
Still not working?

**Comment by slippman** [comment]
@deadrubberboy , based on your comment below, I might be jumping through too many hoops in my solution [here](https://github.com/ollo69/ha-samsungtv-smart/issues/309#issuecomment-2559143460). 

> Thank you @johaywood for the Service Call tip. I confirmed that works. I'll try to make the automation with that! Fingers crossed!

Can you share a few details on how you made the service call work? I couldn't and ended up [creating a work-around](https://github.com/ollo69/ha-samsungtv-smart/issues/309#issuecomment-2559143460). That was possibly a noob not knowing how to make service calls correctly :)

**Comment by deadrubberboy** [comment]
My TV started being an even bigger pain now (unrelated to this integration).  For some unknown reason the CEC suddenly stopped honoring the Apple TV shutdown. It would just hit to “no HDMI signal” instead of turning off or art mode.  

I ended up disabling Anynet+/CEC altogether on the TV and unchecking “control tv” in. Apple TV. This now works 100% with no plug ins or “tech”. TV not even on the network anymore.  Turn on Apple TV w/its remote - TV on. Turn off Apple TV with its remote - Art mode.

**Comment by deadrubberboy** [comment]
Link to YouTube I followed to get it to work:

https://youtu.be/CDENAUxgo6g?feature=shared

**Comment by slippman** [comment]
Unfortunately, that doesnt work on the 2024 Frame TVs. Which is what started me down this rabbit hole in the first place :( That technique works fine on my 2022 Frame TV. Thanks for the share and comment, though! Still would like to see an example of how you called the service in HA for this integration to switch to Art mode!

**Comment by ehn** [comment]
As far as I can tell, the issue has not been resolved and is not stale.

**Comment by Bastian007** [comment]
Yes, I can confirm, the issue is still there.

**Comment by deadrubberboy** [comment]
Confirmed.  Issue still here too.

**Comment by ehn** [comment]
@ollo69 Would it be possible to add one of the `exempt-issue-labels` to this issue to prevent it from being marked as stale? Most of the activity on this issue is just people confirming that the problem remains and the issue shouldn't be considered stale.

**Comment by mjeshurun** [comment]
I have the same issue with my 2019 Frame TV. 
Even though I enabled "Power button switch to art mode (Frame TV only)" in the advance settings, the TV turns completely OFF when pressing the TV's power button within HA.

**Comment by Sammy990** [comment]
As far as I can see from this discussion, there is a new API available for the Frame: https://www.reddit.com/r/TheFrame/s/sAEIcllqE6

Does anyone know if this is already part of this integration?

**Comment by deadrubberboy** [comment]
No change. Not stale

**Comment by ehn** [comment]
As far as I can tell, the issue has not been resolved and is not stale.

**Comment by deadrubberboy** [comment]
Agreed-mSent from my iPhone.On May 27, 2025, at 12:03 AM, Andreas Ehn ***@***.***> wrote:﻿ehn left a comment (ollo69/ha-samsungtv-smart#295)
As far as I can tell, the issue has not been resolved and is not stale.

**Comment by deadrubberboy** [comment]
Not stale

**Comment by manderson0488-oss** [comment]
I was having the same problem. I finally discovered I could go into the smart things app and create an internal automation to trigger when the TV is turned off, turn art mode on.

**Comment by deadrubberboy** [comment]
> I was having the same problem. I finally discovered I could go into the smart things app and create an internal automation to trigger when the TV is turned off, turn art mode on.

I don’t use smart things. Just home assistant/ HomeKit.

**Comment by ehn** [comment]
Not stale

**Comment by deadrubberboy** [comment]
Still fresh

**Comment by ehn** [comment]
Not stale

**Comment by ehn** [comment]
Not stale

**Comment by deadrubberboy** [comment]
Fresh!

**Comment by ehn** [comment]
Not stale

**Comment by deadrubberboy** [comment]
Not stale-mSent from my iPhone.On Mar 27, 2026, at 12:35 AM, github-actions[bot] ***@***.***> wrote:﻿github-actions[bot] left a comment (ollo69/ha-samsungtv-smart#295)
This issue is stale because it has been open 45 days with no activity. Remove stale label or comment or this will be closed in 7 days.

**Comment by ehn** [comment]
Not stale

**Comment by deadrubberboy** [comment]
> This issue is stale because it has been open 45 days with no activity. Remove stale label or comment or this will be closed in 7 days.

Not stale

**Comment by ehn** [comment]
Not stale

**Comment by ehn** [comment]
Not stale
